west.yml

```diff
@@ -0,0 +1,15 @@
+# The west manifest file (west.yml) for the Zephyr Trusted Firmware-M module.
+#
+# This file specifies dependencies of TF-M, as can also be found referenced in
+# the trusted-firmware-m build system.
+#
+# See the west documentation for more information:
+#
+# https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/west/index.html
+
+manifest:
+  projects:
+    - name: tfm-mcuboot
+      url: https://github.com/zephyrproject-rtos/mcuboot
+      path: modules/tee/tfm/mcuboot
```

Comment:
Is it possible that this can be `path: modules/tee/tfm-mcuboot`.  The reason is because of how `west` works have this be a different path from the existing `mcuboot` path makes life a bit easier if one goes back in history w/a checkout.

Reply:
Yes, I think this is reasonable

Reply:
Done